src/map/src/chat/shout_chat.cpp

```diff
@@ -0,0 +1,21 @@
+#include "chat/shout_chat.h"
+
+#include "logconsole.h"
+#include "entity_system.h"
+#include "components/basic_info.h"
+#include "cli_shout_chat.h"
+#include "srv_shout_chat.h"
+
+using namespace RoseCommon;
+using namespace RoseCommon::Packet;
+
+void Chat::shout_chat(EntitySystem& entity_system, Entity entity, const CliShoutChat& packet) {
+    auto logger = Core::CLog::GetLogger(Core::log_type::GENERAL).lock();
+    logger->trace("Chat::shout_chat()");
+    logger->trace("entity {}, message {}", entity, packet.get_message());
```

Comment:
I would have used debug here instead of trace.

Reply:
line 15 only, line 14 would be a trace

Reply:
Yeah, but I check the normal chat and we put the message as a trace instead of debug. So it could be either

Reply:
Updated to a debug message and rebased